src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Verbs.tt

```diff
@@ -91,7 +91,8 @@ foreach (var verb in commands.Descendants(XName.Get("Verb","")))
                 {
 <#
         }
-        foreach (var option in noun.Descendants()) {
+        foreach (var option in noun.Descendants())
```

Comment:
Most of the `var`'s in this file should be replaced with the explicit type.

Reply:
done.